### DISCUSSION TITLE
News Submission 5

### DISCUSSION THREAD

**Discussion by DaveBowman2001** [celestiamobile/celestia-addon-validator#3664] [General]
Continued from [#77](https://github.com/celestiamobile/celestia.mobi/issues/77)

**Comment by DaveBowman2001** [comment]
@levinli303 
HTML: [Week 19, 2024 - Massive Solar Storm Hits Earth on May 10, 2024.txt](https://github.com/celestiamobile/celestia-addon-validator/files/15282158/Week.19.2024.-.Massive.Solar.Storm.Hits.Earth.on.May.10.2024.txt)

# Week 19, 2024: Massive Solar Storm Hits Earth on May 10, 2024
On the second week of May 2024, a series of coronal mass ejections and solar flares originating from a large sunspot group called AR3663, struck the Earth as a powerful geomagnetic storm, with the strongest one so far erupting on May 10th. As a result, it generated aurorae as far south as Florida, USA and as far north as Australia. It is the first G5-class geomagnetic storm observed since 2003 and possibly the strongest one on record since March 1989

**Source**: [Space Weather Live](https://www.spaceweatherlive.com/en/news/view/536/20240511-extreme-g5-geomagnetic-storm.html)


## Gallery of the May 2024 solar storms
![farsidecme_anim_opt](https://github.com/celestiamobile/celestia-addon-validator/assets/95486841/9beb935e-b261-4c3c-83b7-443af7b33e83)
_One of the coronal mass ejections that caused the May 2024 geomagnetic storm as observed by SOHO_

![cover_image](https://github.com/celestiamobile/celestia-addon-validator/assets/95486841/2eef72a5-9c28-4411-afcc-1b1a0e339945)
_AR3663 sunspot group, compared with the 1859 sunspot group that generated the Carrington Event_

![cover_image](https://github.com/celestiamobile/celestia-addon-validator/assets/95486841/4de1b0c6-77eb-4599-91cb-1784742932d8)
_Global aurora forecasts as of May 11, 2024 (Credit: SpaceWeatherLive.com)_

## Aurorae Photographed by Celestia Discord Members
![GridArt_20240511_155200237](https://github.com/celestiamobile/celestia-addon-validator/assets/95486841/6266ef58-a583-472f-8fba-42b1eb9256cd)
_(Credit: Anthony_B_Russo)_

![GridArt_20240511_161319125](https://github.com/celestiamobile/celestia-addon-validator/assets/95486841/71e28705-fe56-48e0-8c5e-f7f483854feb)
_(Credit: arkenar | gnollified)_

![GridArt_20240511_160005109](https://github.com/celestiamobile/celestia-addon-validator/assets/95486841/8508b581-6756-4dfc-9503-b3326d44ab57)
_(Credit: Askaniy)_

![GridArt_20240511_160641641](https://github.com/celestiamobile/celestia-addon-validator/assets/95486841/664a55d8-dd3b-4975-98f3-e52d5faf67e3)
_(Credit: cubicApocalypse)_

![GridArt_20240511_160159766](https://github.com/celestiamobile/celestia-addon-validator/assets/95486841/1fe2fd41-a3ed-498d-bca6-51288b164455)
_(Credit: PlutonianEmpire)_

![GridArt_20240511_160836532](https://github.com/celestiamobile/celestia-addon-validator/assets/95486841/506ac9e5-1aee-416f-b77e-ed9d67678b7f)
_(Credit: Xiphosura)_

**Comment by levinli303** [comment]
> Week 19, 2024: Massive Solar Storm Hits Earth on May 10, 2024

Posted, FYI, I added a link to Astroweather app in this post.

**Comment by DaveBowman2001** [comment]
@levinli303 
HTML: [Week 34, 2024 - Jupiter-bound Spacecraft flies by the Earth and the Moon (August 19–20, 2024).txt](https://github.com/user-attachments/files/16714320/Week.34.2024.-.Jupiter-bound.Spacecraft.flies.by.the.Earth.and.the.Moon.August.19.20.2024.txt)

# Week 34, 2024: Jupiter-bound Spacecraft flies by the Earth and the Moon (August 19–20, 2024)
![IMG_20240820_075058](https://github.com/user-attachments/assets/7cb05994-650c-4ca0-99ce-7b474038ca23)
On 19–20 August 2024, the European Space Agency's *Jupiter Icy Moons Explorer* (JUICE) spacecraft flew past both the Earth and the Moon! Launched in April 2023, the flybys were the first of four planned planetary encounters in order for the mission to gain enough delta-v to reach Jupiter and its moons

JUICE is expected to arrive at Jupiter on July 2031, and will study the planet and its moons concurrently with NASA's *Europa Clipper* mission

The *Jupiter Icy Moons Explorer* is available for download as an addon for Celestia [**here**](https://celestia.mobi/resources/item?item=13D974FA-19C2-9460-C628-A292E83A8BA8)

Source: [European Space Agency](https://www.youtube.com/watch?v=ZoodtR-46Ms)

## Gallery of JUICE's First Earth-Moon Flyby, 19–20 August 2024
![IMG_20240820_075740](https://github.com/user-attachments/assets/c814d153-8c88-474a-9fa0-7a3fe364e725)
![GVYT3NlWUAEldRP](https://github.com/user-attachments/assets/645bdeba-63bf-41c7-b2f0-79cf8e0e6b49)
*JUICE flyby of the Moon, 2024-Aug-19*

![cover_image](https://github.com/user-attachments/assets/a1fc6546-cc4e-415b-a23f-cb55d594bf8e)
*JUICE flyby of the Earth, 2024-Aug-20*

**Comment by levinli303** [comment]
> # Week 34, 2024: Jupiter-bound Spacecraft flies by the Earth and the Moon (August 19–20, 2024)

Posted, should be visible shortly.

**Comment by DaveBowman2001** [comment]
@levinli303
HTML: [Week 35, 2024 - Japan's First Lunar Lander Officially Ends its Historic Mission.txt](https://github.com/user-attachments/files/16843802/Week.35.2024.-.Japan.s.First.Lunar.Lander.Officially.Ends.its.Historic.Mission.txt)

# Week 35, 2024: Japan's First Lunar Lander Officially Ends its Historic Mission
![image](https://github.com/user-attachments/assets/88d50de7-09f5-4672-b988-f9d16eb98a02)
_SLIM about to arrive in lunar orbit, 2023-Dec-25_


Around 22:40 JST (13:40 UTC) of 23 August 2024, the Japanese Aerospace Exploration Agency (JAXA) has officially terminated operations of the ***Smart Lander for Investigating Moon (SLIM)*** mission when it failed to reestablish contact with the probe, three months since its last known transmission with Earth (April 28th)

Launched alongside the XRISM observatory in 6 September 2023, it took more than three months for the probe to arrive on lunar orbit. It made its landing attempt on 19 January 2024 near Shioli Crater, making Japan the fifth country to successfully land on the Moon. Although designed to survive for only one lunar day (14 days), it continued to send back data to Earth for another three months (April 28th) until the spacecraft eventually failed due to the harsh environment of the lunar surface.

The *Smart Lander for Investigating Moon* mission is available for download as an addon for Celestia [**here**](https://celestia.mobi/resources/item?item=AF10A3A1-9609-4F10-BFBC-EED33C20BCAB)

**Source**: [JAXA](https://www.jaxa.jp/press/2024/08/20240826-1_j.html)

## Highlights of the SLIM mission (2023–2024)
![XRISM-Launch](https://github.com/user-attachments/assets/3d432e61-2204-435f-b5bf-b31cf24addd7)
_Launch of XRISM and SLIM aboard the H-IIA rocket, 6 Sept 2023_

![Animation_of_SLIM_around_Earth](https://github.com/user-attachments/assets/729d1fbd-fb8d-4908-81f7-18ed72d336b9)
_Animation of SLIM's trajectory to the Moon, Sept-Dec 2023_

![IMG_4518](https://github.com/user-attachments/assets/9c9fbf77-d124-44ab-8abe-eea123777c9d)
_Craters on the Moon as seen by SLIM in orbit, 2024-Jan-15_

![GEqr4KFaEAEoNKw](https://github.com/user-attachments/assets/dab2f23b-ce18-4804-abc9-b493b32c1e9f)
_SLIM upside down on the lunar surface as seen by LEV-1, 2024-Jan-25_

![chandrayaan-2-slim-lander-24523633-16x9_0](https://github.com/user-attachments/assets/29a84e84-598a-4d3d-9ae0-b261ade889af)
_SLIM landing site (13.316°S 25.251°E) as seen by Chandrayaan-2, 2024-Mar-16_

**Comment by DaveBowman2001** [comment]
@levinli303 
HTML: [Week 36, 2024 - Asteroid Lights Up as a Bright Fireball over the island of Luzon, Philippines (September 4, 2024).txt](https://github.com/user-attachments/files/16888654/Week.36.2024.-.Asteroid.Lights.Up.as.a.Bright.Fireball.over.the.island.of.Luzon.Philippines.September.4.2024.txt)

# Week 36, 2024: Asteroid Lights Up as a Bright Fireball over the island of Luzon, Philippines (September 4, 2024)
![FDownloader Net_An-2R05DDrIt26h_fAl_KPFp73eWg3t5lhfaUMJ3b_L_RtocRZLtDqA6LsGpG18kND63-pPNFEP2WEj9ibRZVSM0_720p_HD-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/edbf19f9-719b-4c3d-aee0-5338397f0484)
_2024 RW₁ lighting up the night sky over Gonzaga, Cagayan (Credit: Allan Madelar)_

On September 4, 2024 a small meteoroid only about 1-m in diameter was discovered to be in a direct collision course with Earth. Designated by the Minor Planet Center (MPC) as **2024 RW₁**, it was first spotted by Jacqueline Fazekas of the _Catalina Sky Survey_ from Mount Lemnon Observatory in Arizona, USA about 14 hours before impact.

It was the ninth known asteroid discovered on a collision course with Earth since 2008. The meteoroid crashed off the coast of the island of Luzon in the Philippines by 00:39 (UTC+8), and despite the stormy weather at the time caused by the departing Typhoon Yagi (Enteng) and the southwest monsoon, many astronomers, space enthusiasts and skywatchers in the country have managed to observe and record the event as it occurs

The asteroid 2024 RW₁ is available for download as an add-on for Celestia **[here](https://celestia.mobi/resources/item?item=D99CED4E-B6EF-47E6-ABBA-BD6EB63D950A)**

**Source:** [Philippine Astronomy Forum](https://www.facebook.com/share/g/ypPheeFU36Y2ZhY9/?mibextid=A7sQZp), [Minor Planet Center](https://minorplanetcenter.net/mpec/K24/K24R68.html), [International Meteor Organization](https://www.imo.net/2024-rw1-atmospheric-entry-in-pictures-videos/)

## Gallery of the Impact of Asteroid 2024 RW₁
![FDownloader net-836547941934580-1080p-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/ae02799a-af0d-491a-99b8-bb504e5975ba)
_Pamplona, Cagayan (Credit: Marvin Coloma)_

![PobPdRm2X63b8vrB-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/9f1a50ff-83db-48e2-a0b5-7d02daf1fc07)
_Tuguegarao City, Cagayan (Credit: Raymon G. Dullona)_

![FDownloader net-928479632424602-1080p-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/dce7a8cf-ee0b-47e2-9bb9-93a5c05f97a0)
_Solana, Cagayan (Credit: Jayrold P. Del Mundo)_

![ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/9cf2c470-0dc3-412d-a82f-0105565d5a7a)
_Santa Ana, Cagayan (Credit: Karen Bea)_

## Asteroid 2024 RW₁ in Context
![89272009-13813111-image-a-60_1725470331054](https://github.com/user-attachments/assets/b82aba04-e695-40c9-aa90-710c30f945a3)
_Discovery images by the Catalina Sky Survey_

![IMG_20240905_075029](https://github.com/user-attachments/assets/0e3a2dd7-b0da-45b3-96d9-1c46f42090f7)
_Predicted impact site as reported by the Minor Planet Center_

![IMG_20240905_080410](https://github.com/user-attachments/assets/19354989-3c82-4e19-816f-39621a66712c)
_Satellite images of the local weather at the time of impact, 00:40 UTC. Impact site was circled in red_

**Comment by DaveBowman2001** [comment]
@levinli303 
HTML File: [Week 41, 2024 - Two Comets Might Become the Brightest Seen from Earth since 1965!.txt](https://github.com/user-attachments/files/17273392/Week.41.2024.-.Two.Comets.Might.Become.the.Brightest.Seen.from.Earth.since.1965.txt)

# Week 41, 2024: Two Comets Might Become the Brightest Seen from Earth since 1965!
The month of October 2024 is an exciting time to be a comet hunter/photographer as there's not only one, but two potentially bright naked eye comets appearing in the night sky!

During the latter half of September 2024, comet **C/2023 A3 (Tsuchinshan-ATLAS)** has become a bright naked eye object in the early morning skies, with many people eagerly anticipating its peak by Oct. 12. Then, shortly after passing perihelion on Sept. 27, the same ATLAS team who discovered this comet has spotted another one, **C/2024 S1 (ATLAS)**, which turned out to be an unusually large Kreutz sungrazer, about to approach the Sun by Oct. 28. Calculations and predictions of both comets indicate that both comets could potentially reach peak magnitudes as high as **-4.8 and -8.4** respectively, and if both remain intact they could become one of the brightest comets ever seen since **Ikeya-Seki** in 1965!

Both **[Tsuchinshan-ATLAS](https://celestia.mobi/resources/item?item=1726BFEB-C111-4A01-9952-6FA0EABBBB9C)** and **[C/2024 S1](https://celestia.mobi/resources/item?item=1EC4AFAF-022A-4461-A271-E9D083DE7D38)** are available for download as add-ons for Celestia

**Sources**: [StarWalk](https://starwalk.space/en/news/new-kreutz-sungrazer-comet-atlas-2024), [Sky & Telescope](https://skyandtelescope.org/astronomy-news/get-ready-for-comet-tsuchinshan-atlas-the-best-is-yet-to-come/)

## C/2023 A3 (Tsuchinshan-ATLAS)
![FB_IMG_1727694266983](https://github.com/user-attachments/assets/08389c2b-2fd2-4fbd-83b7-84b186e3617a)
_San Rafael, Bulacan, Philippines, 2024-Sept-26 (Credit: John Pile)_

![C2023_A3 (1)](https://github.com/user-attachments/assets/f2bcbe2d-4b4f-40b6-9625-c0d525bcfaed)
_Auckland, New Zealand, 2024-Sept-28 (Credit: Alex Liang)_

![FB_IMG_1728276652714](https://github.com/user-attachments/assets/03f4caf6-14ed-4cee-8592-1afafff84da5)
_Tivoli, Namibia, 2024-Oct-2 (Credit: Michael Jäger)_

![FB_IMG_1728082132577](https://github.com/user-attachments/assets/0e533b52-90e0-48a3-ae20-f691b86b48dc)
_Llullailaco Volcano, Chile, 2024-Oct-4 (Credit: Paranal Observatory)_

## C/2024 S1 (ATLAS)
![image](https://github.com/user-attachments/assets/b025909e-36e1-416e-8136-6cae55bd649f)
_Namibia, 2024-Sept-30 (Credit: Michael Jäger)_

![FB_IMG_1728276902484](https://github.com/user-attachments/assets/05f824f1-efad-4db7-bf78-44118e397e27)
_Queensland, Australia, 2024-Oct-6 (Credit: Terry Lovejoy)_

**Comment by DaveBowman2001** [comment]
@levinli303 
HTML File: [Week 17, 2025 - Hubble Celebrates 35 Years in Orbit This Month!.txt](https://github.com/user-attachments/files/19928596/Week.17.2025.-.Hubble.Celebrates.35.Years.in.Orbit.This.Month.txt)

# Week 17, 2025: Hubble Celebrates 35 Years in Orbit This Month!
![Image](https://github.com/user-attachments/assets/82cf7a20-b0ae-493a-b818-f7c505c6281f)
NASA and ESA's **Hubble Space Telescope** celebrates its 35th anniversary in orbit on 24 April 2025, continuing its operations and changing our understanding of the universe as a whole. Launched in 1990 aboard Space Shuttle _Discovery_, it then went on to observe countless celestial objects and beam back thousands of images which we could study and admire here back on Earth. Its longevity is made possible by five Shuttle servicing missions between 1993 and 2009, with a sixth being actively proposed by now-current NASA Administrator, Jared Isaacman, [since 2022](https://www.nasa.gov/missions/hubble/nasa-spacex-to-study-hubble-telescope-reboost-possibility/). It is expected to continue its mission to the cosmos until 2028 or 2040, when atmospheric drag becomes great enough to cause the telescope to reenter the Earth's atmosphere uncontrollably

An improved 3D model of the Hubble Space Telescope can be downloaded as an add-on for Celestia **[here](https://celestia.mobi/resources/item?item=9368F0C1-BF71-FA3D-A521-A13AAA5B84E7)**

**SOURCE**: [NASA](https://science.nasa.gov/mission/hubble/overview/hubbles-35th-anniversary/)

**Comment by DaveBowman2001** [comment]
@levinli303 
HTML File:
[Week 26, 2025 - The Vera C. Rubin Observatory in Chile Releases its First Light Images!.txt](https://github.com/user-attachments/files/20903140/Week.26.2025.-.The.Vera.C.Rubin.Observatory.in.Chile.Releases.its.First.Light.Images.txt)

# Week 26, 2025: The Vera C. Rubin Observatory in Chile Releases its First Light Images!
![Image](https://github.com/user-attachments/assets/bd61e461-cb59-4bcf-8637-4b6243d43ec6)
_The Vera C. Rubin Observatory and the Milky Way Galaxy, 2022_

On 23 June 2025, the Vera C. Rubin Observatory's 8.4-m Large Synoptic Survey Telescope (LSST) has released its first light images as part of its commissioning phase which began on October 2024. The telescope is equipped with the world's largest digital camera ever built, which is capable of producing 3,200 megapixel images which would require at least 400 4K-definition television screens if it were displayed on their original size.

The observatory is expected to produce at least 20 TB of data each night as it observes the entire southern sky for the next 10 years.

**Source**: [Astronomy Magazine](https://www.astronomy.com/science/first-ever-images-released-by-the-vera-c-rubin-observatory/)

## The First Images from the LSST
![Image](https://github.com/user-attachments/assets/c9e425a6-7381-4286-b2e0-7cab97afe892)
_The Trifid and Lagoon Nebulae_

![Image](https://github.com/user-attachments/assets/6e8c914a-09a5-4dbc-aedc-1845fca406a7)
_The Virgo Galactic Cluster_

![Image](https://github.com/user-attachments/assets/64f2d66a-f8d6-4292-a78f-ea4c24091b4f)
_Another view of the Virgo Galactic Cluster_

**Comment by DaveBowman2001** [comment]
@levinli303 
HTML File:
[Week 28, 2025 - Newly-discovered Comet is the Third Interstellar Object Ever Known!.txt](https://github.com/user-attachments/files/21164524/Week.28.2025.-.Newly-discovered.Comet.is.the.Third.Interstellar.Object.Ever.Known.txt)

# Week 28, 2025: Newly-discovered Comet is the Third Interstellar Object Ever Known!
![Image](https://github.com/user-attachments/assets/b25a156d-add8-457a-9873-945b95ac40f2)
_Comet 3I/ATLAS as seen from the Deep Random Survey in Rio Hurtado, Chile on 2 July 2025_

Last week on 1 July 2025, the _Asteroid Terrestrial Last-Alert System_ (ATLAS) survey in Chile discovered a new interstellar comet (**A11pl3Z**) passing through the Solar System. The object, now known as **3I/ATLAS**, is the third known after 'Oumuamua and Borisov

As of 2 July 2025, the comet is currently about 4.00 AU (600 million km) from the Earth, with an orbital eccentricity of 6.0 and moving at approximately 68 km/s relative to the Sun.

3I/ATLAS is available in default content since 10 July 2025 and an improved model is available for download for Celestia **[here](https://celestia.mobi/resources/item?item=D8442CC8-0D2C-42B6-BD42-959BD2AB0580)**

**Source**: [NASA](https://science.nasa.gov/solar-system/comets/3i-atlas/)

**Comment by DaveBowman2001** [comment]
Post under construction
# Celestia Celebrates its 25th Anniversary on February 26, 2026!

**Comment by levinli303** [comment]
> Post under construction
> 
> # Celestia Celebrates its 25th Anniversary on February 26, 2026!

im planning to run a promotion for discounted Celestia PLUS around the week too, might go along with the post.

**Comment by DaveBowman2001** [comment]
@levinli303 
HTML File: [Week 13, 2026 - Artemis II Launches on the First Crewed Mission to the Moon since 1972.txt](https://github.com/user-attachments/files/26420420/Week.13.2026.-.Artemis.II.Launches.on.the.First.Crewed.Mission.to.the.Moon.since.1972.txt)

# Week 13, 2026: Artemis II Launches on the First Crewed Mission to the Moon since 1972!
<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/37d91c1e-2955-4ba9-9c3b-ddbcb8f5d4d8" />
_Celestia simulation of Artemis II during its closest approach to the Moon, 2026-Apr-6_

Around 22:24 UTC on Wednesday, 1 April 2026, NASA successfully launched **Artemis II** on a 10-day mission to the Moon. It is the first crewed spaceflight to the Moon since Apollo 17 in December 1972, ending a 54-year gap in manned spaceflight beyond LEO.

The mission will make a flyby around the Moon at a distance of 6,000 km on 6 April 2026. By the time, they will be around 407,000 km from Earth, breaking the record of the furthest distance travelled by humans set by Apollo 13 in 1970. No landing attempt is planned until **Artemis IV** in 2028. 

All Artemis missions are available for download as add-ons for Celestia [**HERE**](https://celestia.mobi/resources/item?item=A2437ED3-5A19-425F-B95F-BE251D9ECD97)

## Official Livestream
```
<iframe width="560" height="315" 
  src="https://www.youtube.com/live/Tf_UjBMIzNo" 
  frameborder="0" allowfullscreen>
</iframe>
```

## Crew
<img width="800" height="1000" alt="Image" src="https://github.com/user-attachments/assets/929f2aee-cc6b-475b-be65-a5a9f1dfcf90" />
_Official portrait of the crew of Artemis II (annotated)_

Christina Koch, Victor Glover and Jeremy Hansen will become the first woman, first person of color, and first non-American astronauts to travel to the Moon respectively

## Mission Trajectory
<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/39a0b0e8-2ab1-430f-b7a5-13d918e5c248" />
_Celestia simulation of Artemis II's prelaunch trajectory (retrieved from JPL-HORIZONS)_

**Comment by levinli303** [comment]
> Around 22:24 UTC on Wednesday, 1 April 2026, NASA successfully launched **Artemis II** on a 10-day mission to the Moon. It is the first crewed spaceflight to the Moon since Apollo 17 in December 1972, ending a 54-year gap in manned spaceflight beyond LEO.

this is posted, note that Ive dropped the "Week 13, 2026" in the title as we are no longer posting weekly. and we are showing Publish date already.
https://celestia.mobi/resources/guide?guide=63B1B825-7FD1-42BF-8EF2-30A553A5364F

**Comment by DaveBowman2001** [comment]
@levinli303 
HTML File: [Artemis II makes its Closest Approach to the Moon.txt](https://github.com/user-attachments/files/26522349/Artemis.II.makes.its.Closest.Approach.to.the.Moon.txt)

# Artemis II makes its Closest Approach to the Moon
![Image](https://github.com/user-attachments/assets/b70868cb-ea50-4ef4-a1fb-8172160c7c24)
_The Earth about to set down the lunar horizon as seen from NASA's [official livestream](https://www.youtube.com/watch?v=z-j1uxBmis0) of Artemis II's lunar flyby_

The crew of Artemis II has now achieved the furthest distance away from Earth during their flyby of the Moon, breaking the record previously held by Apollo 13 in April 1970!

On Monday, 6 April 2026 (roughly about 23:00 UTC) the Orion spacecraft passed by the lunar surface at a distance of 6,500 km while they are approximately 407,000 km away from Earth. A communications blackout occurred just as expected, as the Earth set over the lunar horizon, lasting over 40 minutes as they begin their journey back to Earth. During their lunar observations, the crew named a crater on the Far Side of the Moon after the late wife of Commander Reid Wiseman, Carroll

While on their way, the crew of Artemis II are even greeted by the pre-recorded messages of two Apollo astronauts, Charlie Duke and Jim Lovell respectively. Lovell recorded his message shortly before his death in August 2025.

All Artemis missions are available for download as add-ons for Celestia **[HERE](https://celestia.mobi/resources/item?item=A2437ED3-5A19-425F-B95F-BE251D9ECD97)**

## Official NASA Livestream of the Lunar Flyby
```
<h3>Official NASA Livestream of the Lunar Flyby</h3>
<iframe width="560" height="315" 
  src="https://www.youtube.com/watch?v=z-j1uxBmis0" 
  frameborder="0" allowfullscreen>
</iframe>
```
## Images from the Lunar Flyby
![Image](https://github.com/user-attachments/assets/1bb0d488-143c-4340-b0c5-2980a76ccdd0)
<img width="1668" height="763" alt="Image" src="https://github.com/user-attachments/assets/42683b85-96de-459f-98c6-4990bca1aac5" />
![Image](https://github.com/user-attachments/assets/4e36331d-fd7c-46ef-8c3d-df3180ea599c)
![Image](https://github.com/user-attachments/assets/37cce3a8-c341-4974-9c0a-6e73d4c85228)
![Image](https://github.com/user-attachments/assets/31d70b42-f8df-43fa-a217-86de21a67e04)

**Comment by levinli303** [comment]
> [@levinli303](https://github.com/levinli303) HTML File: [Artemis II makes its Closest Approach to the Moon.txt](https://github.com/user-attachments/files/26522349/Artemis.II.makes.its.Closest.Approach.to.the.Moon.txt)

this is published, I've set the notification time until April 11, 12:00:00 AM UTC (after this time, it will not be shown to users who haven't seen it on app launch)
